Cesium.Runtime/StdLibFunctions.cs

```diff
@@ -7,6 +7,7 @@ public unsafe static class StdLibFunctions
 {
     public static int Abs(int value)
     {
+        if (value == int.MinValue) return int.MinValue;
         return Math.Abs(value);
     }
```

Comment:
Wow that's a gimmick. Let's write a unit test for the standard library, then?

Reply:
Maybe we can, but I mostly think that should do that as part of "acceptance testing", but in retrospection after I think a bit on this I do agree with you. That's subsystem should be tested.